build/configs/artik05x/artik05x_download.sh

```diff
@@ -20,37 +20,13 @@
 # File   : artik05x_download.sh
 # Description : Download script for ARTIK 05X
 
-# Remember, make is invoked from "os" directory
 source .config
-OS_DIR_PATH=${PWD}
-BUILD_DIR_PATH=${OS_DIR_PATH}/../build
-CONFIGS_DIR_PATH=${BUILD_DIR_PATH}/configs
-OUTPUT_BINARY_PATH=${BUILD_DIR_PATH}/output/bin
-OPENOCD_DIR_PATH=${BUILD_DIR_PATH}/tools/openocd
-ARTIK05X_DIR_PATH=${CONFIGS_DIR_PATH}/artik05x
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
```

Comment:
You don't use "SCRIPT_DIR". Let's remove this or use it at below line.

Reply:
sure, i'll do it shall be the same as in *ota.sh in same dir.